src/components/GlobalPreferences/Network/Network.js

```diff
@@ -53,7 +53,7 @@ function Network({ wrapper }) {
           )}
         </Label>
         <Label>
-          IPFS Gateway{' '}
+          IPFS Gateway
```

Comment:
If you remove `{' '}` then the text will appear right next to the input. Either we use the `{' '}` (usually `prettier` adds this) or set a `margin` in the input

Reply:
When the `TextInput` is wide I don't think this happens:

<img width="748" alt="Screen Shot 2019-08-16 at 2 59 46 PM" src="https://user-images.githubusercontent.com/4166642/63169235-8b56cf80-c036-11e9-869b-03590bb14aaa.png">

Reply:
Jajaja, unless there is no need for the space when we want the text and input in different line blocks.. :+1